install_config/registry/extended_registry_configuration.adoc

```diff
@@ -186,20 +186,20 @@ openshift:
     secret: <secret>
 ----
 
-. Create a new secret called *registry-config* from your custom registry
-configuration file you edited locally:
+. Create a `*ConfigMap*` holding the content of each file in this directory:
 +
 ----
-$ oc secrets new registry-config config.yml=</path/to/custom/registry/config.yml>
+$ oc create configmap registry-config \
+    --from-file=</path/to/custom/registry/config.yml>/
 ----
 
-. Add the *registry-config* secret as a volume to the registry's deployment
+. Add the *game-config* ConfigMap as a volume to the registry's deployment
```

Comment:
The `game-config` does not correspond to the `registry-config` in `oc create configmap registry-config` few lines above. I find `registry-config` better.